test-programs/account-compression-test/tests/address_merkle_tree_tests.rs

```diff
@@ -187,7 +202,7 @@ async fn test_insert_invalid_low_element() {
         bigint_to_be_bytes_array(&address1).unwrap(),
         bigint_to_be_bytes_array(&address2).unwrap(),
     ];
-    insert_addresses(
+    let _ = insert_addresses(
```

Comment:
would it make sense to unwrap twice instead of putting a result or option into an placeholder variable?

Reply:
Yes, thanks, I cleaned it up.